state_city.js

```diff
@@ -0,0 +1,43 @@
+
+const COUNTRY_STATE_CITY_DD = { 
+    "india": {
+      "gujarat": ["Surat", "Rajkot", "Jamnagar", "Vadodara"],
+      "up": ["Kanpur", "Agra", "Noida", "Saharanpur"],
+      "dehli": ["Gokalpuri", "New-Dehli", "Mandoli", "Taj-Pul"]
+    },
+    "usa": { 
+      "texas": ["Austin", "SanAntonio", "Dallas", "Houston"],
+      "chicago": ["Lyons", "ForestPark", "RiverForest", "Lincolnwood"],
+      "lasVegas": ["SpringValley", "Boulder", "Paradise", "Henderson"]
+    }
+  };
+
+function addOption(dropdown, array){
+  for (var index = 0; index < array.length; index++) {
+    var option = document.createElement("option");
+    option.text = array[index].toUpperCase();
+    dropdown.add(option);
+  }
+  return dropdown;
+}
+function getCountryValue(){
+    let country_dropdown = document.getElementById("country").value.toLowerCase();
+    var state_dropdown = document.getElementById("states");
```

Comment:
Why you are using `var` here when you have used `let` in below. Also, you are not accessing the value here after the `getElementById("states")` whereas in below you are using `.value` after this. Please take time to review your code before you asked review from side. So you can find your mistakes on your own.